README.md

```diff
@@ -1,46 +1,35 @@
-# Sway Visual Studio Code Plugin
+<picture>
+   <source media="(prefers-color-scheme: light)" srcset="./images/light/sway.png">
```

Comment:
let's wait until we get the updated images from Grant before landing this PR. They look really cool though!

Reply:
updated!